modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_2_0/DDMFacetTemplateUpgradeProcessTest.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.internal.upgrade.v5_2_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * @author Yuri Monteiro
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class DDMFacetTemplateUpgradeProcessTest {
```

Comment:
There are more integration tests than unit tests in DDM (see dynamic-data-mapping-test). 
It might be easier to write this as an integration test since we won’t need to mock anything.
Could you convert it to an integration test?